holidays/countries/syrian_arab_republic.py

```diff
@@ -0,0 +1,141 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.constants import GOVERNMENT, OPTIONAL, PUBLIC
+from holidays.groups import InternationalHolidays, IslamicHolidays, StaticHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class SyrianArabRepublic(HolidayBase, InternationalHolidays, IslamicHolidays, StaticHolidays):
+    """
+    Syrian Arab Republic holidays.
+
+    References:
+    * [Timeanddate.com](https://www.timeanddate.com/holidays/syria/2025)
+    * [OfficeHolidays.com](https://www.officeholidays.com/countries/syria/2025)
+    * [PublicHolidays.me](https://publicholidays.me/syria/2025-dates/)
+    * [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Syria)
+
+    Holidays are regulated by the Syrian government and may vary each year.
+    """
+
+    country = "SY"
+    default_language = "ar"
+    estimated_label = tr("(تقدير) %s")
+    supported_categories = (GOVERNMENT, OPTIONAL, PUBLIC)
+    supported_languages = ("ar", "en_US")
+    start_year = 1946  # Syria's independence
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SyrianArabRepublicIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        StaticHolidays.__init__(self, SyrianArabRepublicStaticHolidays)
+        super().__init__(*args, **kwargs)
+
+    def _add_christmas_day(self):
+        """Add Christmas Day to the list of holidays."""
+        self._add_holiday("Christmas Day", 12, 25)
+
+    def _populate_public_holidays(self):
+        # New Year's Day
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
+
+        # Revolution Day
+        self._add_holiday_mar_8(tr("عيد الثورة"))
+
+        # Mother's Day
+        self._add_holiday_mar_21(tr("عيد الأم"))
+
+        # Evacuation Day (Independence Day)
+        self._add_holiday_apr_17(tr("عيد الجلاء"))
+
+        # Labor Day
+        self._add_labor_day(tr("عيد العمال"))
+
+        # Martyrs' Day
```

Comment:
Make sure the name used here matches the sources you've mentioned i.e. https://cb.gov.sy/index.php?lang=2&dir=html&ex=1&page=holidays&act=598-

"October Liberation War" -> "Anniversary of the October Liberation War"